interface/main/calendar/add_edit_event.php

```diff
@@ -447,6 +467,21 @@ function setEventDate($start_date, $recurrence)
      *                    UPDATE EVENTS
      * =====================================================*/
     if ($eid) {
+
+        /**
+         * @todo make this an event that is dispatched with these variables
+         */
+        if ($_POST['form_apptstatus'] === '>') {
+            $bInfo = add_escape_custom($_POST['form_title']);  //calendar appointment type billing information
+            $pid = add_escape_custom($_POST['form_pid']);      //patient id
+            $userid = add_escape_custom($_SESSION['authUserID']);   //user who entered the information
```

Comment:
remove the add_escape_custom() wrappers above (note you are properly sql escaping them via binding downstream)